frontend/lib/utils/cornerstone3DInit.ts

```
@@ -0,0 +1,912 @@
+let initialized = false;
+
+const { ViewportType } = Enums;
+const { MouseBindings } = ToolEnums;
+
+// Create a compatibility layer to handle API differences between versions
+const compat = {
+  // Safe method to initialize DICOM image loader
+  initializeDicomImageLoader: async (): Promise<void> => {
+    try {
+      // Try to set up cornerstone reference if needed (older versions)
+      if (dicomImageLoaderLib.hasOwnProperty('external')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)